configs/default.yaml

```diff
@@ -21,40 +25,51 @@ dataset:
         rescale_size: [144, 256]
     responses:
       keep_nans : False
-      sampling_rate: null
-      chunk_size: null
-      offset: 0.1 # in seconds
+      sampling_rate: 8
+      chunk_size: 16
+      offset: 0.0 # in seconds
       transforms:
         normalization: "standardize"
       interpolation:
         interpolation_mode: "nearest_neighbor"
+      filters:
+        nan_filter:
+          __target__: experanto.filters.common_filters.nan_filter
+          __partial__: True
+          vicinity: 0.05
     eye_tracker:
       keep_nans : False
-      sampling_rate: null
-      chunk_size: null
+      sampling_rate: 30
+      chunk_size: 60
       offset: 0
       transforms:
         normalization: "normalize"
       interpolation:
         interpolation_mode: "nearest_neighbor"
+      filters:
+        nan_filter:
+          __target__: experanto.filters.common_filters.nan_filter
+          __partial__: True
+          vicinity: 0.05
     treadmill:
       keep_nans : False
-      sampling_rate: null
-      chunk_size: null
+      sampling_rate: 30
+      chunk_size: 60
       offset: 0
       transforms:
         normalization: "normalize"
       interpolation:
         interpolation_mode: "nearest_neighbor"
+      filters:
+        nan_filter:
+          __target__: experanto.filters.common_filters.nan_filter
+          __partial__: True
+          vicinity: 0.05
```

Comment:
I would suggest to rename it in sth line `vicinity_in_sec` to make it clear that vicinity is measured in seconds and its the only place we use wall time in the config